migrations/helper/ENSHelper.js

```diff
@@ -61,10 +61,58 @@ const getENS = async (artifacts, network) => {
   }
 };
 
+const deployLocalReverseRegistrar = async (deployer, moderator, artifacts, resolver) => {
+  const ENS = artifacts.require('ENSRegistry');
+  const ReverseRegistrar = artifacts.require('ReverseRegistrar');
+
+  const ens = await ENS.deployed();
+
+  console.log('Deploying Reverse Registrar on ganache');
+  await deployer.deploy(ReverseRegistrar, ens.address, resolver.address);
+  const reverseRegistrar = await ReverseRegistrar.deployed();
+
+  await ens.setSubnodeOwner('0x0000000000000000000000000000000000000000', utils.sha3('reverse'), moderator);
+  await ens.setSubnodeOwner(
+    namehash.hash('reverse'),
+    utils.sha3('addr'),
+    reverseRegistrar.address,
+    { from: moderator }
+  );
+};
+
+const reverseRegister = async (account, name, artifacts, network) => {
+  const ENS = artifacts.require('ENSRegistry');
+  const ReverseRegistrar = artifacts.require('ReverseRegistrar');
+  const RinkebyReverseRegistrar = require('./RinkebyReverseRegistrar');
+
+  let reverseRegistrar;
+
+  switch (network) {
+  case 'development':
+  case 'test':
+  case 'soliditycoverage':
+  case 'ganache':
+    reverseRegistrar = await ReverseRegistrar.deployed();
+    break;
+  case 'rinkeby':
+  case 'rinkeby-fork':
+    const ens = await ENS.at(ENS_RINKEBY);
+    const registrarAddress = await ens.owner(namehash.hash('addr.reverse'));
+    reverseRegistrar = await new web3.eth.Contract(RinkebyReverseRegistrar, registrarAddress);
+    break;
```

Comment:
Mix and matching between web3 and truffle contracts here. Probably not a big issue just something to be aware of.